salt/modules/win_file.py

```diff
@@ -818,8 +818,10 @@ def stats(path, hash_type='sha256', follow_symlinks=True):
 
         salt '*' file.stats /etc/passwd
     '''
+    # This is to mirror the behavior of file.py. `check_file_meta` expects an
+    # empty dictionary when the file does not exist
```

Comment:
I tend to wonder if this should be changed in the other direction. Wouldn't it be cleaner to raise a specific error and then check for it and display it to the user, versus just an empty dict?

Reply:
@cachedout Changed this back, added try/except block to calling function